lib/user.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const gravatar = require('gravatar');
+
+module.exports = class User {
+    constructor({ name, email, imageUrl }) {
+        this.email = email;
+        this.name = name;
```

Comment:
Missing validation on name.

Reply:
The idea was to make it optional. It feels a bit overkill to have the user specify `name` and `email` when using the builtin cookie based session. We are only using the email to build gravatar and tag changes.  

What do you think?

Reply:
Indeed optional, but could pick it up anyhow. Currently the builtin does not pass it along.

Suggestion: We could also default the name from the email, e.g.`/^(.*)@/i`